packages/model-viewer/src/features/ar.ts

```diff
@@ -340,9 +349,50 @@ configuration or device capabilities');
       const img = document.createElement('img');
       anchor.appendChild(img);
       anchor.setAttribute('href', modelUrl.toString());
+      if (this.arAutogenerateUsdz) {
+        anchor.setAttribute('download', 'model.usdz');
+      }
       anchor.click();
       anchor.removeChild(img);
     }
+
+    [$onModelLoad]() {
+      super[$onModelLoad]();
+      if (this.arAutogenerateUsdz && !!this.iosSrc) {
+        URL.revokeObjectURL(this.iosSrc);
```

Comment:
model-viewer can have its src changed, so after new model is being loaded we should release the old URL so it can be generate again next time user wants to go to Quick Look